serverless-multiregion.yml

```diff
@@ -1,62 +1,80 @@
 include:
-  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.2.5/templates/serverless.yml'
+  - remote: 'https://raw.githubusercontent.com/jobtome-labs/ci-templates/v2.3.0/serverless-quality.yml'
 
 # EUROPE
 deploy:production:europe:
   extends: .serverless:deploy
   variables:
-    GOOGLE_KEY: ${GOOGLE_KEY_EUROPE}
-    REGION: ${REGION_EUROPE}
-    DOMAIN: ${DOMAIN_EUROPE}
+    GOOGLE_KEY: ${GOOGLE_KEY_PRODUCTION_EUROPE}
+    SERVICE_ACCOUNT: ${SERVICE_ACCOUNT_PRODUCTION_EUROPE}
+    REGION: ${REGION_PRODUCTION_EUROPE}
+    DOMAIN: ${DOMAIN_PRODUCTION_EUROPE}
     ENVIRONMENT: production-europe
-    SECRET_YAML: ${SECRET_YAML_EUROPE}
-    SECRET_ENV_LIST: ${SECRET_ENV_LIST_EUROPE}
+    TRIGGER_TOPIC: ${TRIGGER_TOPIC_PRODUCTION_EUROPE}
+    SECRET_YAML: ${SECRET_YAML_PRODUCTION_EUROPE}
+    SECRET_ENV_LIST: ${SECRET_ENV_LIST_PRODUCTION_EUROPE}
   before_script:
     - |
       # CHECK VARIABLES PHASE
-      for var in "GOOGLE_KEY_EUROPE" "REGION_EUROPE"; do
+      for var in "GOOGLE_KEY_PRODUCTION_EUROPE" "REGION_PRODUCTION_EUROPE"; do
```

Comment:
remove _PRODUCTION_EUROPE, you defined the "simple name" (GOOGLE_KEY) above

Reply:
I've defined `GOOGLE_KEY: ${GOOGLE_KEY_PRODUCTION_EUROPE}` because I want one key for one environment